stable/zeppelin/templates/zeppelin-configmap.yaml

```diff
@@ -52,5 +52,7 @@ data:
     fi
 
     /docker-zeppelin.sh \
+{{- if .Values.spark }}
             --master-hostname={{ .Values.spark.hostname }} \
```

Comment:
I don't think the `docker-zeppelin.sh` script can operate without these params.

Reply:
Tested both with spark and without. without spark it logs that it expect spark details but it launches zeppelin, so its not really a must